E/Exafmmt/build_tarballs.jl

```diff
@@ -0,0 +1,63 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Exafmmt"
+version = v"0.1.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JoshuaTetzner/exafmm-t.git", "359558a6ae7b82ea936f44a33400fa6ca7e27e52")
+]
+
+#/workspace/x86_64-linux-gnu-libgfortran5-cxx11/artifacts/9f61595dd5f6597db74a071ac26c2d094b563029/lib/libblastrampoline.so
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd exafmm-t/
+mkdir build
+cd build
+BLAS=blastrampoline
+LAPACK=blastrampoline
```

Comment:
You don't have libblastrampoline in your dependencies.  Instead, you have OpenBLAS and LAPACK which I don't understand what they're used for.